hadoop-tools/hadoop-azure/src/site/markdown/index.md

```diff
@@ -18,8 +18,8 @@
 
 See also:
 
-* [ABFS](./abfs.html)
-* [Testing](./testing_azure.html)
+* [ABFS](./abfs.md)
```

Comment:
-1. when site docs are generated *.md is mapped to *.html.

Reply:
Reverting this change.
But I observed these links are not working on github.
https://github.com/apache/hadoop/blob/trunk/hadoop-tools/hadoop-azure/src/site/markdown/index.md

They are giving 404

Reply:
run mvn site:site and you generate the site locally, which is what goes up on apache.org